src/System.Net.Mail/tests/Functional/SmtpServer.cs

```diff
@@ -77,7 +77,8 @@ public void Run()
             }
         }
 
-        // return false == terminate
+        public bool Support_SMTPUTF8 { get; set; }
```

Comment:
I would bump this up to the top of the class (ie, around lines 25-29) so that anyone who looks at this code immediately sees all of the properties.

Reply:
This should be renamed to conform to style guidelines.  I.e. something like "SupportsSmtpPutf8"